styles.css

```
@@ -139,19 +141,14 @@ nav .right-content a:hover {
     justify-content: center
 }
 
-.text-button{
-    font-size: 21px;
-    font-weight: bolder;
-}
-
 #container-buttons a{
     text-decoration: none;
     color: #000000;  
 }
 
-a :hover{
-    background-color: #F3F3F3;
-    transition: all 1s linear;
+#container-buttons a:hover{
+    background-color: transparent;
+    transition: 1s;
```

Muito bem @jacqueline-abreu.
A velocidade da animacao esta principalmente relacionada com a propriedade `1s`. Ela quer dizer que a animacao vai durar 1 segundo. Dei uma breve verificada no template e animacao deles dura 0.4 segundos.
Entao acho que a sua mudanca deveria ser algo assim:
```suggestion
    transition: all 0.4s linear;
```

---

Fiz a modificação solicidada. Realmente ficou mais parecido. Na modificação anterior eu tinha notado uma aceleração quando retirei o "all" e "linear", mas acho que pode ter sido uma impressão errada de tanto olhar pra animção.